### ISSUE TITLE
您好, 嘗試重製ernie-vil-VCR-task pretrained model 在 VCR task validation split 的結果, 表現不理想 

### ISSUE THREAD

**Issue by Mallory24** [PaddlePaddle/ERNIE#649]
ERNIE團隊您們好,

目前拿公開的ernie-vil-VCR-task-pretrained model (large & base) 預訓練好的參數, 重製VCR task的結果, 但表現遠低於版上的結果:
1. model-ernie-vil-large-VCR-task-pre-en 在 VCR-QA validation split 僅得 avg acc: 23.5
2. model-ernie-vil-base-VCR-task-pre-en 在 VCR-QA validation split 僅得 avg acc: 43.9

嘗試找出問題原因, 但還是有幾個問題想請教:
1. 從results.txt 結果實際算acc, large可得 65, base 可得 67, 檢查代碼似乎沒問題
![acc](https://user-images.githubusercontent.com/20028769/116684968-ef0f6080-a9b1-11eb-92ca-73fd48eceb84.png)

2. 檢查results.txt 結果, 每個qid multiple choices (4) 的 softmax scores 都差不多落在 0.25, 推測提供的預訓練模型參數有些問題, 否則應該對multiple choices 問題的選擇更有自信

希望您們能提供一些方向解決無法重製實驗結果的問題, 感謝您們

**Comment by yinweichong** [comment]
ERNIE-ViL (task-pretrain) base & large是使用vcr数据进行task pretrain的模型，不是finetune 的模型。在VCR任务上的效果，需要使用task pretrain的模型进行finetune，之后再inference

**Comment by Mallory24** [comment]
了解, 感謝你們的回覆!
最後, 想確認一下, 所謂的VCR task pretrain是指用Masked Language Modeling (MLM) 和 Masked Region Prediction  (MRP) 兩個pretraining objectives 在 VCR train set 上 pretrain嗎?  
(paper上寫你們follow UNITER 的 second stage task pretrain, 所以我想應該是這個意思)

**Comment by yinweichong** [comment]
是的

**Comment by Tclz** [comment]
我也遇到了这个问题！利用model-ernie-vil-base-VCR-task-pre-en仅能得到26%+的acc。感谢！

**Comment by Tclz** [comment]
你好，我在finetune过程中，在已经跑了20个epochs的情况下，模型的acc依旧不理想。控制台打印输出如下：
epoch: 21, progress: 0/0, step: 26620, loss: 0.551648, acc : 0.375000
20210621 14:41:47 current learning_rate:0.00000020
used_time: 0.631230354309082
feed_queue size 26
epoch: 21, progress: 0/0, step: 26630, loss: 0.541245, acc : 0.375000
20210621 14:41:53 current learning_rate:0.00000020
used_time: 0.6923425197601318
feed_queue size 30
epoch: 21, progress: 0/0, step: 26640, loss: 0.538330, acc : 0.437500
20210621 14:42:00 current learning_rate:0.00000020
save_path: output_vcr/step_26640
used_time: 41.171826124191284

均是使用的本仓库提供的配置文件，没有改动。我使用的finetune命令如下：
`CUDA_VISIBLE_DEVICES=0 sh run_finetuning.sh vcr conf/vcr/model_conf_vcr model-ernie-vil-base-VCR-task-pre-en/vocab.txt model-ernie-vil-base-VCR-task-pre-en/ernie_vil_config.base.json model-ernie-vil-base-VCR-task-pre-en/params/* result/base-finetune.txt`
是我的打开方式不对么？求大佬解答

**Comment by Tclz** [comment]
> 了解, 感謝你們的回覆!
> 最後, 想確認一下, 所謂的VCR task pretrain是指用Masked Language Modeling (MLM) 和 Masked Region Prediction (MRP) 兩個pretraining objectives 在 VCR train set 上 pretrain嗎?
> (paper上寫你們follow UNITER 的 second stage task pretrain, 所以我想應該是這個意思)

你好，请问你最后复现出了finetune以及inference的结果吗？
